base/settings/develop.py

```diff
@@ -1,12 +1,13 @@
-from .base import *
+import os
+from .base import *  # noqa
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
 ALLOWED_HOSTS = ['*']
 
-INSTALLED_APPS += [
-]
+INSTALLED_APPS += [  # noqa
+]  # noqa
```

Comment:
Is this setting necessary if you are not adding something?

Reply:
Good catch! As I added an explanation in the commit comment.  Add '# noqa' tag for specific lines which make flake8 fail but they actually are correct.
Flake8 suggests that INSTALLED_APPS could be not defined but as I imported the base settings it always will be defined.